### PR TITLE
docs(prisma): use prisma migrate instead of instrospection

### PR DESCRIPTION
**Pull request by nikolasburk**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

The docs for the Prisma recipe now use the new Prisma Migrate (which will be released tomorrow, Tuesday, Dec 8) instead of introspection.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: content/recipes/prisma.md [hunk 1]
```diff
@@ -537,4 +530,9 @@ This controller implements the following routes:
 
 In this recipe, you learned how to use Prisma along with NestJS to implement a REST API. The controller that implements the routes of the API is calling a `PrismaService` which in turn uses Prisma Client to send queries to a database to fulfill the data needs of incoming requests.
 
-If you want to learn more about Prisma, be sure to check out the [documentation](https://www.prisma.io/docs/).
+If you want to learn more about using NestJS with Prisma, be sure to check out the following resources:
+
+- [NestJS & Prisma](https://wwww.prisma.io/nestjs)
+- [Ready-to-run example projects for REST & GraphQL]()
```
Comment (johannesschobel) [inline content/recipes/prisma.md:536]: An URI is missing here

**Review by johannesschobel — COMMENTED** [review]
Would it be a good idea to reference the current version of Prisma this guide is referencing? i.e., this guide uses Prisma 2.x?

File: content/recipes/prisma.md (hunk 1 above)
Comment (nikolasburk) [inline content/recipes/prisma.md:536]:
```suggestion
- [Ready-to-run example projects for REST & GraphQL](https://github.com/prisma/prisma-examples/)
```

Comment (nikolasburk) [reply content/recipes/prisma.md]: Hey Johannes, nice to see you around again 😄 thanks for catching this, just added the URL pointing to the `prisma-examples` repo 👍

**Comment by nikolasburk** [comment]
> Would it be a good idea to reference the current version of Prisma this guide is referencing? i.e., this guide uses Prisma 2.x?

Personally I think this is not necessary as Prisma 2 really is the default (and it's actually just called "Prisma"). Everything that's about Prisma 1.x should always be labeled as "Prisma 1" explicitly. 

However, if you prefer adding a note about this I'm totally open to adding it somewhere on the page 🙂

File: content/recipes/prisma.md (hunk 1 above)
Comment (johannesschobel) [reply content/recipes/prisma.md]: Thanks for the welcoming words ;)

**Review by johannesschobel — APPROVED** [review]

**Comment by nikolasburk** [comment]
Since [Prisma Migrate has been released yesterday](https://www.prisma.io/blog/prisma-migrate-preview-b5eno5g08d0b), this page should be ready to get merged 🙌

File: content/recipes/prisma.md [hunk 2]
```diff
@@ -537,4 +530,9 @@ This controller implements the following routes:
 
 In this recipe, you learned how to use Prisma along with NestJS to implement a REST API. The controller that implements the routes of the API is calling a `PrismaService` which in turn uses Prisma Client to send queries to a database to fulfill the data needs of incoming requests.
 
-If you want to learn more about Prisma, be sure to check out the [documentation](https://www.prisma.io/docs/).
+If you want to learn more about using NestJS with Prisma, be sure to check out the following resources:
+
+- [NestJS & Prisma](https://wwww.prisma.io/nestjs)
```
Comment (kamilmysliwiec) [inline content/recipes/prisma.md:535]:
```suggestion
- [NestJS & Prisma](https://www.prisma.io/nestjs)
```

**Comment by kamilmysliwiec** [comment]
LGTM. Thanks!

**Pull request merged**